Defs/ResearchProjectDefs/LotRE_Research.xml

```diff
@@ -0,0 +1,147 @@
+
+  <ResearchProjectDef ParentName="ElvenResearch">
+    <defName>LotRE_ArmorTwo</defName>
+    <label>Buckler and Armor Smithing</label>
+    <description>Prepare your Elves with armor and helmets for the horrors of war:\n- Falathrim-Elf Armor and Helmet\n- Silvan-Elf Armor and Helmet\n- Elf Buckler</description>
+    <baseCost>250</baseCost>
+    <prerequisites>
+      <li>LotRE_ArmorOne</li>
+    </prerequisites>
+    <researchViewX>0.75</researchViewX>
+    <researchViewY>1</researchViewY>
+  </ResearchProjectDef>
+
+  <ResearchProjectDef ParentName="ElvenResearch">
+    <defName>LotRE_ArmorThree</defName>
+    <label>Armor of the Eldar</label>
+    <description>Protect your Elves with apparel of incredible elven quality:\n- Galadhrim-Elf Armor and Helmet</description>
+    <baseCost>500</baseCost>
+    <prerequisites>
+      <li>LotRE_ArmorTwo</li>
+    </prerequisites>
+    <researchViewX>1.5</researchViewX>
+    <researchViewY>1</researchViewY>
+  </ResearchProjectDef>
+
+  <ResearchProjectDef ParentName="ElvenResearch">
+    <defName>LotRE_ArmorFour</defName>
+    <label>Armor of the Ñoldor</label>
+    <description>Inlay, shape, and produce armor with stunning unrivaled quality worthy of a true Children of Ilúvatar :\n- Ñoldor-Elf Armor, Helmet, and Shield\n- Elf Crown</description>
+    <baseCost>800</baseCost>
+    <prerequisites>
+      <li>LotRE_ArmorThree</li>
+    </prerequisites>
+    <researchViewX>2.25</researchViewX>
+    <researchViewY>1</researchViewY>
+  </ResearchProjectDef>
+
+  <!--======== Column 3 =========-->
+  <!--======= Elves Furniture ======-->
+
+  <ResearchProjectDef ParentName="ElvenResearch">
+    <defName>LotRE_FurnitureOne</defName>
+    <label>Elven Furniture</label>
+    <description>Elven furniture is just as elegant as Elves themselves:\n- Elven Table (2x2)\n- Elven Chair\n- Elven Lounge Bed</description>
+    <baseCost>250</baseCost>
+    <tags>
+      <li>LotRE_ElfScenario</li>
+    </tags>
+    <researchViewX>0</researchViewX>
+    <researchViewY>2</researchViewY>
+  </ResearchProjectDef>
+
+  <ResearchProjectDef ParentName="ElvenResearch">
+    <defName>LotRE_FurnitureTwo</defName>
+    <label>Elven Lighting</label>
+    <description>Unlock an Elven candelabra for providing light in dark places:\n- Elven candelabra</description>
+    <baseCost>150</baseCost>
+    <prerequisites>
+      <li>LotRE_FurnitureOne</li>
+    </prerequisites>
+    <researchViewX>0.75</researchViewX>
+    <researchViewY>1.7</researchViewY>
+  </ResearchProjectDef>
+
+  <ResearchProjectDef ParentName="ElvenResearch">
+    <defName>LotRE_FurnitureThree</defName>
+    <label>Regal Elven Furniture</label>
+    <description>Furniture of such high quality it could only be Elven made:\n- Elven royal bed</description>
+    <baseCost>250</baseCost>
+    <prerequisites>
+      <li>LotRE_FurnitureOne</li>
+    </prerequisites>
+    <researchViewX>0.75</researchViewX>
+    <researchViewY>2.3</researchViewY>
+  </ResearchProjectDef>
+
```

Comment:
All these additions to Elven research look solid. I appreciate the attention to detail and mirroring of the Dwarves' research tab in style.